playground/frontend/playground_components/lib/src/widgets/run_button.dart

```diff
@@ -73,8 +73,8 @@ class RunButton extends StatelessWidget {
                     )
                   : const Icon(Icons.play_arrow),
               label: isRunning
-                  ? PeriodicRebuilderWidget(
-                      rebuildFrequencyMls: 1,
+                  ? PeriodicBuilderWidget(
+                      interval: const Duration(milliseconds: 1),
```

Comment:
1 millisecond is too heavy. 100 milliseconds? Anyway extract to a constant in this file.